apps/server/src/controllers/user.ts

```diff
@@ -58,15 +58,15 @@ export const loginUser = async (req: Request, res: Response) => {
   try {
     const user = await User.findOne({ username: req.body.username });
     if (!user) {
-      return res.status(401).json({ error: 'Username not found' });
+      return res.status(409).json({ error: 'Username not found' });
```

Comment:
I think 400 might more suitable for this according to this (open to disucussion😅)
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409